doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,103 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
+
+1. Using the `kexec` tool as described in link:../../usr/share/rear/conf/templates/RESULT_usage_RAMDISK.txt[RESULT_usage_RAMDISK.txt],
+   if you have access to a running Linux instance.
+
+2. Using the `zipl` tool for management of boot devices on s390/s390x.
+
+=== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM with `zipl`
+
+After creating the rescue image with `rear mkrescue`, move the kernel and
+initramfs to the target directory (denoted by `REAR_DIRECTORY` in this example)
```

Comment:
I think using the term 'target directory' here
it is still misleading/confusing because
REAR_DIRECTORY is the "source" directory
where zipl gets kernel and initrd from
while the target is BOOTMAP_INSTALL_DIRECTORY